docs/configuration/index.md

```diff
@@ -329,6 +329,20 @@ wheel.exclude = ["**.pyx"]
 Previously these were matched on the source path, rather than the wheel path,
 and didn't apply to CMake output.
 
+You can exclude files from the built wheel based on file extension as well (not
+guaranteed to be respected by editable installs):
+
+```toml
+[tool.scikit-build]
+wheel.exclude-exts = [".c", ".cuh", ".h"]
+```
+
+Default value: `[".pyc", ".pyo"]`
+
+:::{versionadded} 0.12
+
+Support to exclude based on file extension.
```

Comment:
I don't think the format looks right here, can you rearrange it? https://scikit-build-core--1128.org.readthedocs.build/en/1128/configuration/index.html#customizing-the-built-wheel

Also, this is new since the introduction of `Config Reference`, but can you use `:confval:...` role to reference the new option? There's a nice pop-up of the information about it, so you could de-duplicate some documentation with that.